src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset>
+
+        <legend>Personal information</legend>
+
+        <label class="block">
+          Surname:
+          <input type="text" required
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="block">
+          Name:
+          <input type="text" autocomplete="off">
+        </label>
+         
+        <label class="block">
+          How old are You? 
+          <input type="number" value="12"
+           min="1" max="100"
+          >
+        </label>
+
+        <label class="block">
+          Full date of birth: 
+          <input type="date" required>
+        </label>
+
+        <label class="block">
+          I accept the term of the agreement
+          <input type="checkbox" required>
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+
+        <legend>Registration</legend>
+
+        <label class="block">
+          E-mail:
+          <input type="email" placeholder="email@example.com"
+           name="e-mail" required
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="block">
+          Password:
+          <input type="password" name="password"
+           required autocomplete="off"
+          >
```

Comment:
If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.
![image](https://user-images.githubusercontent.com/61904995/220125678-68528ca7-f291-4719-a394-6aaf79ba9dcc.png)